lib/omnibus/fetchers/net_fetcher.rb

```diff
@@ -125,6 +125,11 @@ def checksum
       source[digest_type]
     end
 
+    # Set to false to extract archives into the root source_dir (default).
+    # Set to true to always extract underneath project_dir
+    #
+    attr_accessor :isolate_extraction
```

Comment:
if this does change that behavior, won't this essentially break backwards compat? (Not that I'm against that, just need to make sure we version and discuss correctly)